test/test_other.py

```diff
@@ -10042,6 +10042,7 @@ def test_minimal_runtime_code_size(self, test_name, js, compare_js_output=False)
                                '-sMIN_CHROME_VERSION=58',
                                '-sGL_WORKAROUND_SAFARI_GETCONTEXT_BUG=0',
                                '-sNO_FILESYSTEM',
+                               '-sSTRICT',
```

Comment:
Why add this? I'd think the opposite is better as it's closer to what most users do.

Reply:
This test is all about specifying as many non-default settings as possible in order to minimize code size.   While adding `-sSTRICT` here doesn't currently shave any bytes of the code size one could imagine that it could reasonable do this (imagine a case where we need to include legacy support for something, but we can avoid doing that in strict mode).

Adding this setting here triggers all the code paths where the old name (USE_PTHREADS) was incorrectly used internally (but ignored because of we allow the old name to be used as an alias by default).

In other words, this change is what actually tests that I've fixed the bug.

Reply:
I see. Sounds good!